landing_page_app/main/scripts/github_script.py

```diff
@@ -19,64 +16,37 @@ class GithubScript:
     def __init__(self, github_service: GithubService):
         self.github_service = github_service
 
-    def _check_email_address(self, email_address: str):
-        valid = False
-        if email_address is not None and email_address != "":
-            regex = re.compile("[@]")
-            if regex.search(email_address) is not None:
-                valid = True
-        return valid
-
-    def _is_email_address_pre_approved(self, organisation: str, email_address: str):
-        pre_approved = False
-        if (
-            self._check_email_address(email_address)
-            and organisation != ""
-            and organisation is not None
-        ):
-            email_domain = email_address[email_address.index("@") + 1:]
-            if organisation.lower() == MOJ_ANALYTICAL_SERVICES:
-                if email_domain in AS_ORG_ALLOWED_EMAIL_DOMAINS:
-                    pre_approved = True
-            elif organisation.lower() == MINISTRY_OF_JUSTICE:
-                if email_domain in MOJ_ORG_ALLOWED_EMAIL_DOMAINS:
-                    pre_approved = True
-        return pre_approved
+    def add_returning_user_to_github_org(self, username: str, organisations: list):
+        if username == "" or username is None or len(organisations) == 0:
+            logger.debug(
+                "add_returning_user_to_github_org: incorrect function argument"
+            )
```

Comment:
Think it's worth throwing an error here so the user is redirected to an error page?

Reply:
The function before this function check the input parameters are correct:

if current_app.github_script.validate_user_rejoining_org(selected_orgs, username):

Reply:
which redirects to the error screen:

return error(
    "Username not found or has expired. Create a new request and leave the username box empty."
)